tests/gdbstub.sh

```diff
@@ -11,20 +11,31 @@ then
         -ex "file build/puzzle.elf" \
         -ex "target remote :1234" \
         -ex "break *0x10700" \
+        -ex "break *0x10800" \
+        -ex "break *0x10900" \
+        -ex "continue" \
+        -ex "print \$pc" \
+        -ex "continue" \
+        -ex "print \$pc" \
         -ex "continue" \
         -ex "print \$pc" \
         -ex "del 1" \
+        -ex "del 2" \
+        -ex "del 3" \
         -ex "stepi" \
         -ex "stepi" \
         -ex "continue" > ${tmpfile}
 
     # check if we stop at the breakpoint
-    expected=$(grep -rw "Breakpoint 1 at" ${tmpfile} | awk {'print $4'})
-    ans=$(grep -r "$1 =" ${tmpfile} | awk {'print $5'})
-    if [ "$expected" != "$ans" ]; then
-        # Fail
-        exit 1
-    fi
+    for i in {1..3}
+    do
+        expected=$(grep -rw "Breakpoint ${i} at" ${tmpfile} | awk {'print $4'})
```

Comment:
Bash array is powerful. Check it out: https://opensource.com/article/18/5/you-dont-know-bash-intro-bash-arrays
You can utilize this feature to shorten the above.

Reply:
Do you mean using bash array with element `(1 2 3)` here? Otherwise, I don't find any better place to make use of it.